lnst/Recipes/ENRT/SRIOVNetnsBridgeRecipe.py

```diff
@@ -0,0 +1,187 @@
+        dict(gro="off", gso="on", tso="on", tx="on", rx="on"),
+        dict(gro="on", gso="off", tso="off", tx="on", rx="on"),
+        dict(gro="on", gso="on", tso="off", tx="off", rx="on"),
+        dict(gro="on", gso="on", tso="on", tx="on", rx="off")))
+
+    def test_wide_configuration(self):
+        """
+        Test wide configuration for this recipe involves switching the device to switchdev
+        mode, adding single VF and mapping the VF, as well as its representor.
+        New namespace is created to mimic container networking, where the VF is moved.
+        Next, VF is assigned an IPv4 and IPv6 address on both hosts.
+        Finally, new Linux bridge is created, where PF and VF representor are added, to
+        ensure correct switching of network traffic between the hosts.
+
+        host1.eth0 = 192.168.101.1/24 and fc00::1/64
+
+        host2.eth0 = 192.168.101.2/24 and fc00::2/64
+        """
+        host1, host2 = self.matched.host1, self.matched.host2
+        configuration = super().test_wide_configuration()
+        configuration.test_wide_devices = []
+
+        for i, host in enumerate([host1, host2]):
+            host.run(f"devlink dev eswitch set pci/{host.eth0.bus_info} mode switchdev; sleep 2")
+            host.run(f"echo 1 > /sys/class/net/{host.eth0.name}/device/sriov_numvfs; sleep 5")
+
+            vf_ifname = dict(ifname=f"{host.eth0.name}v0")
+            host.map_device("vf_eth0", vf_ifname)
+
+            host.newns = NetNamespace(f"lnst{i + 1}")
+            host.newns.vf_eth0 = host.vf_eth0
+
+            vf_representor_ifname = dict(ifname="eth0")
+            host.map_device("vf_representor_eth0", vf_representor_ifname)
+
+            host.br0 = BridgeDevice()
+            host.br0.slave_add(host.eth0)
+            host.br0.slave_add(host.vf_representor_eth0)
+
+            for dev in [host.eth0, host.newns.vf_eth0, host.vf_representor_eth0, host.br0]:
+                dev.up()
+
+            host.newns.vf_eth0.ip_add(ipaddress("192.168.101." + str(i + 1) + "/24"))
+            host.newns.vf_eth0.ip_add(ipaddress("fc00::" + str(i + 1) + "/64"))
+
+            configuration.test_wide_devices.append(host.newns.vf_eth0)
+
+        self.wait_tentative_ips(configuration.test_wide_devices)
+
+        return configuration
+
+    def generate_test_wide_description(self, config):
+        desc = super().generate_test_wide_description(config)
+        host1, host2 = self.matched.host1, self.matched.host2
+        for i, host in enumerate([host1, host2]):
+            desc += [
+                f"Configured {host.hostid}.{host.eth0.name}.driver = switchdev\n"
+                f"Created virtual function on {host.hostid}.{host.eth0.name} = {host.newns.vf_eth0.name}\n"
+                f"Created network_namespace on {host.hostid} = {host.newns.name}\n"
+                f"Moved interface {host.newns.vf_eth0.name} from {host.hostid} to {host.hostid}.{host.newns.name}\n"
```

Comment:
I expect this would look like:
```
Moved interface some_name from host1 to host1.lnst1
```

This is a bit confusing as it does not mention "namespace". I'd change this to:
```
Moved interface some_name from host1 root namespace to lnst1 namespace.
```

Reply:
Fixed